backend/src/ploneconf.core/src/ploneconf/core/behaviors/session.py

```diff
@@ -29,6 +29,7 @@ class IConferenceSession(model.Schema):
             vocabulary="ploneconf.core.vocabularies.persons",
         ),
         required=False,
+        default=[],
```

Comment:
As long as I remember using dict / list in default was always a bad idea since the list is instantiated during plone startup.

But maybe this is not relevant in this usecase if the attribute on plone object is overrided when add or edit.

Reply:
Not in the case of Volto and restapi it seems. The initial (on adding) is an empty array, but then it gets saved empty. This is a known issue in restapi, I tried to fix it, but it's quite complex. A lot of things going in dexterity that fixes all this use cases. On edit, then the value is null, then Volto errors. I'm preparing a PR in Volto to catch that up. However, I think that in a Volto only environment we should be ok.